chemprop/uncertainty/predictor.py

```diff
@@ -91,12 +113,42 @@ def _calc_prediction_uncertainty(
 
 @UncertaintyPredictorRegistry.register("dropout")
 class DropoutPredictor(UncertaintyPredictor):
-    def _calc_prediction_uncertainty(
-        self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer
-    ) -> Tensor:
-        ...
-        return
-
+    """
+    Class that creates an artificial ensemble of models by applying monte carlo dropout to the loaded
+    model parameters. Predicts uncertainty for predictions based on the variance in predictions among
+    an ensemble's submodels.
+    """
+
+    def __init__(self, sampling_size: int, dropout_p: float):
+        self.sampling_size = sampling_size
+        self.dropout_p = dropout_p
```

Comment:
```suggestion
    sampling_size: int
    """The number of samples to draw for the ensemble."""
    dropout_p: float
    """The probability of dropping out units in the dropout layers."""
```

Reply:
Upon testing, I realized that after this change, there's no way to set the dropout probability and sampling size of dropout predictors without directly changing the class variables. Additionally, they are undefined by default. Was this intended?

Reply:
Oops! Let's stick with the original design.